### PR TITLE
Modified import  of `mat_struct` for deprecation warning

### PR DESCRIPTION
**Pull request by h-mayorquin**
Small PR that brings the number of warnings in this repo to 0!
: D

### CODE REVIEW

**Comment by h-mayorquin** [comment]
Ok, so the problem is Python 3.7, I fixed with a conditional import that fetches the right attribute when it exists and gets it from the location in the scipy version that gets installed with Python 3.7 when not.

File: src/roiextractors/extraction_tools.py [hunk 1]
```diff
@@ -13,7 +14,10 @@
 except ImportError:
     HAVE_H5 = False
 try:
-    import scipy.io as spio
+    if hasattr(scipy.io.matlab, "mat_struct"):
```
Comment (CodyCBakerPhD) [inline src/roiextractors/extraction_tools.py:17]: I, for one, very much appreciate that Python modules treat submodules as attributes.

**Review by CodyCBakerPhD — APPROVED** [review]

**Comment by CodyCBakerPhD** [comment]
@h-mayorquin Very nice, thanks! Would this be necessary as well on NWBCT (like Neuroscope/CellExplorer Interfaces?)

**Comment by h-mayorquin** [comment]
> @h-mayorquin Very nice, thanks! Would this be necessary as well on NWBCT (like Neuroscope/CellExplorer Interfaces?)

I will take a loot into it.

**Pull request merged**